mathics/doc/common_doc.py

```diff
@@ -596,6 +734,196 @@ def __str__(self) -> str:
         return f"== {self.title} ==\n{self.doc}"
 
 
+class DocTests:
+    """
+    Object to hold a sequence of related DocTest objects for a section or
+    subsection.
+    Access ``tests`` to get these.
+    """
+
+    def __init__(self):
+        self._tests = []
+        self.text = ""
+
+    @property
+    def tests(self) -> list:
+        """
+        Retrieves test items of this DocTests object.
+        """
+        return self._tests
+
+    # Deprecate
+    def get_tests(self):
+        """
+        Older form of ``.tests`` attribute. Don't use.
+        """
+        print("Warning: get_tests is deprecated")
+        return self._tests
+
+    def is_private(self) -> bool:
+        return all(test.private for test in self.tests)
+
+    def __str__(self) -> str:
+        return "\n".join(str(test) for test in self.tests)
+
+    def test_indices(self) -> List[int]:
+        return [test.index for test in self.tests]
+
+
+class DocSection:
```

Comment:
The class is duplicated here.

Reply:
This code had been fixed locally. It should be addressed in next push.